replica/database.cc

```diff
@@ -1928,7 +1928,7 @@ future<> database::do_apply(schema_ptr s, const frozen_mutation& m, tracing::tra
         }
     }
 
-    sync = sync || db::commitlog::force_sync(s->wait_for_sync_to_commitlog());
+    sync = sync || db::commitlog::force_sync(s->static_props().wait_for_sync_to_commitlog);
```

Comment:
You could have kept s->wait_for_sync_to_commitlog(), just make it look at static_props(). This doesn't necessarily have to come only from static_props(), so would be more future-proof, and shorter.

Reply:
ok